# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/robin-hood-hashing/cstdint.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Fix the build with GCC 16: include <cstdint> for the fixed width integer types.

--- robin-hood-hashing-3.11.5.orig/src/include/robin_hood.h	2022-01-14 18:43:03.000000000 +0100
+++ robin-hood-hashing-3.11.5/src/include/robin_hood.h	2026-09-27 18:34:04.581449086 +0200
@@ -40,6 +40,7 @@
 
 #include <algorithm>
 #include <cstdlib>
+#include <cstdint>
 #include <cstring>
 #include <functional>
 #include <limits>
